Source/WebKit/WebKitSwift/WritingTools/WKIntelligenceTextEffectCoordinator.h

```diff
@@ -34,17 +34,18 @@
 #import <WebKit/_WKTextPreview.h>
 #endif
 
+#import <WebKit/WKWebView.h>
```

Comment:
Is this important necessary, or can you forward declare here?

Reply:
it’s needed since Swift does not understand forward declaration types